components/search_results_item/__snapshots__/search_results_item.test.tsx.snap

```diff
@@ -17,12 +17,21 @@ exports[`components/SearchResultsItem should hide reply button when said so 1`]
         "create_at": 1502715365009,
         "delete_at": 0,
         "edit_at": 1502715372443,
+        "hashtags": "",
```

Comment:
Snapshots had to be updated because post objects were not complying with PostType. Had to add some missing attributes